docs/source/about/guides/linux.rst

```diff
@@ -0,0 +1,459 @@
+	* `Nvidia Documentation on Configuring TwinView <https://download.nvidia.com/XFree86/Linux-x86/270.29/README/configtwinview.html>`_
+	* `Arch Wiki Nvidia#TwinView <https://wiki.archlinux.org/title/NVIDIA#TwinView>`_
+	* `Unix Stack Exchange - How to add virtual display monitor with Nvidia proprietary driver <https://unix.stackexchange.com/questions/559918/how-to-add-virtual-monitor-with-nvidia-proprietary-driver>`_
+
+
+Uinput Permissions Workaround
++++++++++++++++++++++++++++++
+
+**Steps**
+
+We can use ``chown`` to change the permissions from a script. Since this requires ``sudo``, we will need to update the sudo configuration to execute this without being prompted for a password.
+
+#. Create a ``sunshine-setup.sh`` script to update permissions on ``/dev/uinput``. Since we aren't logged into the host, the udev rule doesn't apply.
+#. Update user sudo configuration ``/etc/sudoers.d/<user>`` to allow the ``sunshine-setup.sh`` script to be executed with ``sudo``.
+
+.. admonition:: Why is this necessary?
+	:class: important
+
+	After I setup the :ref:`udev rule <about/usage:linux>` to get access to ``/dev/uinput``,
+	I noticed when I sshed into the host without physical login, the ACL permissions on ``/dev/uinput`` were not changed.
+	So I asked `reddit <https://www.reddit.com/r/linux_gaming/comments/14htuzv/does_sshing_into_host_trigger_udev_rule_on_the/>`_.
+	I discovered that SSH sessions are not the same as a physical login.
+	I suppose it's not possible for SSH to trigger a udev rule.
+
+**Setup Script**
+
+This script will take care of any precondtions prior to starting up sunshine.
+Create a script named something like ``sunshine-setup.sh``:
+
+.. code-block:: bash
+
+	#!/bin/bash
+	chown <user>:<user> /dev/uinput
+
+	# Optional
+	# blocks wifi, so ethernet is used
+	# use rfkill list to get the id of the Wiresless LAN
+	# rfkill block <wireless_lan_index>
+
+**Sudo Configuration**
+
+We will manually change the permissions of ``/dev/uinput`` using ``chown``.
+You need to use ``sudo`` to make this change, so add/update the entry in ``/etc/sudoers.d/<user>``
+
+.. caution::
+	Do so at your own risk! It is more secure to give sudo and no password prompt to a single script, than a generic executable like chown.
+
+.. warning::
+	Be very careful of messing this config up. If you make a typo, *YOU LOSE THE ABILITY TO USE SUDO*. Fortunately, your system is not borked,
+	you will need to login as root to fix the config. You may want to setup a backup user / SSH into the host as root to fix the config if this happens. Otherwise you will need to plug your machine back into a monitor and login as root to fix this. To enable root login over SSH edit your SSHD config, and add ``PermitRootLogin yes``, and restart the SSH server.
+
+Replace ``<user>`` with your user and add the absolute path to your script:
+
+.. code-block::
+
+	<user> ALL=(ALL:ALL) ALL, NOPASSWD: /path/to/sunshine-setup.sh
```

Comment:
```suggestion
.. code-block:: bash

	echo "${USER} ALL=(ALL:ALL) ALL, NOPASSWD: $(pwd)/sunshine-setup.sh" | sudo tee -a /etc/sudoers.d/${USER}
```

Haven't tested this, but should work to just run from terminal. https://linuxize.com/post/linux-tee-command

Reply:
cool! i will test this

Reply:
I don't think appending would work correctly. it would create something lke this:
```
eric ALL=(ALL:ALL) ALL
eric ALL=(ALL:ALL) ALL, NOPASSWD: /home/eric/sunshine-setup.sh
```
There should also be a note to `cd` to the dir where the `sunshine-setup.sh` lives so the `$(pwd)` will give the correct path.